app/controllers/reports/regions_controller.rb

```diff
@@ -63,10 +67,17 @@ def details
       [@region, @region.facility_regions].flatten
     end
     @repository = Reports::Repository.new(regions, periods: @period_range)
+    chart_repo = Reports::Repository.new(@region, periods: chart_range)
```

Comment:
We could just use `@repository` too I think. It includes the `@region`'s report and `@period_range` is a superset of `chart_range`. Will make the controller easier to follow.

Reply:
I tried this, and the problem is that the range of data we need for the charts is much larger than the data we need for the tables (24 months vs 6).  So if we use the single repository, we end up pulling back a bunch more data for the children regions than we need.

This added at least 500 ms to a large district in my dev testing for the cached case, so pretty bad perf hit for regions w/ lots of data.